Packs/FireEyeETP/Integrations/FireEyeETPEventCollector/FireEyeETPEventCollector.py

```diff
@@ -439,6 +444,11 @@ def get_events(self, event_type: EventType, last_run: LastRun) -> tuple[LastRun,
             case _:
                 raise DemistoException("Event's type format is undefined.")
 
+        if event_type.name != "activity_log":
+            direction = "outbound" if event_type in OUTBOUND_EVENT_TYPES else "inbound"
+            for evt in events:
+                evt["direction_source"] = direction
+
```

Comment:
Its probably affect less the performance if we do it in the fetch_email_trace and in the fetch_alerts functions with lambda.